doc/nrf-bm/sample/intro/create_sample.rst

```diff
@@ -1,5 +1,7 @@
-Creating a copy of the sample
-=============================
+.. _sample_intro_copy:
+
+Creating a copy of a sample
+###########################
 
 Follow these steps to create a new application based on this sample:
```

Comment:
The wording "this sample" seems wrong here.
```suggestion
Follow these steps to create a new application based on a sample:
```
"this sample" is used one more time further down in this file and should be fixed.